nginx/datadog_checks/nginx/nginx.py

```diff
@@ -416,3 +417,7 @@ def _normalize_tags_type(self, tags, device_name=None, metric_name=None):
                     continue
                 normalized_tags.extend(list({tag, self.degeneralise_tag(tag)}))
         return normalized_tags
+
+    def list_stream_endpoints(self, api_dict_list):
```

Comment:
```suggestion
    def list_endpoints(self, api_dict_list):
```
nit since it's not just stream endpoints

Reply:
Edited. 🙇